source/_integrations/roborock.markdown

```diff
@@ -278,10 +420,10 @@ target:
 
 Roborock servers require accepting a user agreement before using the API, which may block Home Assistant during setup. Additionally, the Roborock may ask you to re-enter the user agreement, even if you have entered it before.  To allow Home Assistant to use the Roborock API, you need to take the following steps:
 1. Open your Roborock app.
-2. Open **Profile** > **About Us** > **User Agreement & Privacy Policy**.
-3. Hit **Revoke authorization**.
-4. Log back in and accept the policy.
-5. Reload the Roborock integration!
+1. Open **Profile** > **About Us** > **User Agreement & Privacy Policy**.
+1. Hit **Revoke authorization**.
+1. Log back in and accept the policy.
+1. Reload the Roborock integration!
```

Comment:
```suggestion
2. Open **Profile** > **About Us** > **User Agreement & Privacy Policy**.
3. Hit **Revoke authorization**.
4. Log back in and accept the policy.
5. Reload the Roborock integration!
```
did you see this numbering style in the examples somewhere? If so, the example should be fixed. I think the style is still "numbers increasing".